dimod/constrained/constrained.py

```diff
@@ -688,6 +688,118 @@ def add_discrete_from_model(self,
         self.discrete.add(label)
         return label
 
+    def add_linear_constraints(self, A: ArrayLike, sense: Union[Sense, str], b: ArrayLike,
+                               *,
+                               variable_labels: Optional[Iterable[Variable]] = None,
+                               constraint_labels: Optional[Iterable[Hashable]] = None,
+                               ):
+        r"""Add multiple linear constraints to the model.
+
+        Add linear constraints defined by
+
+        .. math::
+
+                A_{ub}x \le b_{ub}, \\
+                A_{eq}x = b_{eq}
+
+        where `x` is a vector of decision variables,
+        :math:`b_{ub}` and :math:`b_{eq}` are vectors,
+        and :math:`A_{ub}` and :math:`A_{eq}` are matrices.
+
+        Args:
+            A: Each row of ``A`` specifies the coefficients of a linear constraint.
+            sense: One of `<=`, `>=`, `==`.
+            b: An array of left-hand-sides of the constraints.
+            variable_labels: The variable labels corresponding to the columns of `A`.
+                If ``None``, defaults to ``range(A.shape[1])``.
+            constraint_labels: The labels of the constraints. Labels of additional
+                constraints are generated.
```

Comment:
Do you mean here something like, "Labels are generated for any unlabeled constraints"?